SampleProject/SampleProject/Scenes/Favorites/Entity/FavoritesRecipeCellModel.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  FavoritesRecipeCellModel.swift
+//  SampleProject
+//
+//  Created by Mehmet Salih Aslan on 27.07.2022.
+//  Copyright © 2022 Mobillium. All rights reserved.
+//
+
+class FavoritesRecipeCellModel: RecipeSmallCellProtocol {
+
+    let recipeId: Int
+    let userId: Int
+    let userImageUrl: String?
+    let username: String?
+    let recipeTitle: String?
+    let categoryName: String?
+    let recipeImageUrl: String?
+    var commentCount: Int
+    var likeCount: Int
+    let isEditorChoice: Bool
+    var followButtonTapped: VoidClosure?
+    
+    lazy var recipeCommnetAndLikeCountText: String? = {
```

Comment:
Why need to use lazy var? both count variables were declared with var,

     var commentCount: Int
     var likeCount: Int

and here is the corner case:  
if the count will change after the text was created, the string will look like the wrong count just because of lazy var.

I'm not sure your requirements here but, it's looking like a problematic

Reply:
both **commentCount**  and **likeCount** both updated to **let**.